apps/ledger-live-mobile/src/components/Web3AppWebview/PlatformAPIWebview.tsx

```diff
@@ -336,19 +336,19 @@ export const PlatformAPIWebview = forwardRef<WebviewAPI, WebviewProps>(
                 provider,
               },
               onResult: (result: {
-                startExchangeResult?: string;
-                startExchangeError?: Error;
+                startExchangeResult?: { nonce: string; device: Device };
+                startExchangeError?: { error: Error; device?: Device };
                 device?: Device;
               }) => {
```

Comment:
Here we should use the types already present instead of redeclaring them, I tested that locally too
```ts
              onResult: result => {
```